concepts/classes/about.md

```diff
@@ -0,0 +1,260 @@
+
+Often, you want all the derived objects to include some fields and pass some initial values for those when the object is constructed.
+This is facilitated via the [`this` keyword][mdn-this].
+Inside the constructor function, `this` represents the new object that will be created via `new`.
+`this` is automatically returned from the constructor function when it is called with `new`.
+
+That means we can add fields to the new instance by adding them to `this` in the constructor function.
+
+```javascript
+function Car(color, weight) {
+  this.color = color;
+  this.weight = weight;
+  this.engineRunning = false;
+}
+
+const myCar = new Car('red', '2mt');
+myCar.color;
+// => 'red'
+myCar.engineRunning;
+// => false
+```
+
+### Instance Methods
+
+Methods are added via the `prototype` property of the constructor function.
+Inside a method, you can access the fields of the instance via `this`.
+This works because of the following general rule.
+
+> When a function is called as a method of an object, its `this` is set to the object the method is called on. [^1]
+
+```javascript
+function Car() {
+  this.engineRunning = false;
+  // ...
+}
+
+Car.prototype.startEngine = function () {
+  this.engineRunning = true;
+};
+
+Car.prototype.addGas = function (litre) {
+  // ...
+};
+
+const myCar = new Car();
+myCar.startEngine();
+myCar.engineRunning;
+// => true
+```
+
+### The Prototype Chain
+
+`myCar` in the example above is a regular JavaScript object and if we would inspect it (e.g. in the browser console) we would not find a property `startEngine` with a function as value directly inside the `myCar` object.
+So how does the code above even work then?
+
+The secret here is called the _prototype chain_.
+When you try to access any property (field or method) of an object, JavaScript first checks whether the respective key can be found directly in the object itself.
+If not, it continues to look for the key in the object referenced by the `[[prototype]]` property of the original object.
+As mentioned before, in our example `[[prototype]]` points to the `prototype` property of the constructor function.
+That is where JavaScript would find the `startEngine` function because we added it there.
+
+And the chain does not end there.
+The `[[prototype]]` property of `Car.prototype` (`myCar.[[prototype]].[[prototype]]`) references `Object.prototype` (the `prototype` property of the `Object` constructor function).
+It contains general methods that are available for all JavaScript objects, e.g. `toString()`.
+In conclusion, you can call `myCar.toString()` and that method will exist because JavaScript searches for that method throughout the whole prototype chain.
+You can find a detailed example in the [MDN article "Inheritance and the prototype chain"][mdn-prototype-chain-example].
+
+Note that the prototype chain is only travelled when retrieving a value.
+Setting or deleting a property of an instance object only targets that specific instance.
```

Comment:
"Unlike most class-based inheritance" or something similar? I think we should point out the differences to what people are used with.

Perhaps also changes to _setting a property directly or deleting a property_, because you _can_ set properties via the prototype chain, but it's convoluted (requires a method that sets it, effectively).

Reply:
Added the first part you suggested as a separate sentence as it was hard to squeeze in.